app/models/decidim/votings/voting.rb

```diff
@@ -35,11 +35,11 @@ def in_scope?(_user)
       end
 
       def started?
-        start_date.to_datetime < DateTime.current
+        start_date < Time.zone.now
```

Comment:
I guess that this is related to the timezone time that we were discussing, but I don't get it. What is the difference with `Time.now`?

Reply:
In this case there is no difference, because we are comparing times with times, but when extracting day info, Time.zone.now will return the current day taking into account the timezone. As we are working with TimeWithZone in the attrubutes, working with TimeWithZone in the code will avoid unnecessary castings.